example/ExampleDialect.td

```diff
@@ -268,3 +268,36 @@ def HandleGetOp : ExampleOp<"handle.get", [Memory<[]>, NoUnwind, WillReturn]> {
     Use a dialect type without type arguments.
   }];
 }
+
+def InstNameConflictOp : Op<ExampleDialect, "try.conflict", [WillReturn]> {
+  let results = (outs I32:$result);
+  let arguments = (ins value:$instName);
+
+  let summary = "demonstrate how name conflict will be avoided";
+  let description = [{
+    The builder accepts an additional argument to set the name of the created
+    value like IRBuilder methods. This op produces a conflict so the parameter
+    will be renamed.
+  }];
+}
+
+def InstNameConflictDoubleOp : Op<ExampleDialect, "try.conflict", [WillReturn]> {
```

Comment:
Ugh... defining two ops with the same mnemonic should be caught by TableGen. It's definitely not correct.

Reply:
Oh, that was not the conflict I intended to test for. Fixed the conflict and reported #86